.ddev/local.config.php.dist

```diff
@@ -52,10 +52,10 @@ return [
     'version'           => 'OAuth1a',
 
     // Required for OAuth1a and OAuth2
-    'baseUrl'           => 'http://localhost/mautic/index_dev.php',
+    'baseUrl'           => 'http://localhost/mautic/index.php',
```

Comment:
@mabumusa1 why specify `/index.php` and not `/`?
I see no benefit doing so (here and in all other files

Reply:
@mollux There is no benefit, I just removed them. Thanks for the feedback on this, if it is good let's merge it

Reply:
@mollux I just checked the workflow settings again, we do not have `mod_rewrite` enabled, so we need to explicitly specify `index.php`

Reply:
@mabumusa1 can you push?

Reply:
@escopecz I just did, when CI is green you can merge